### ISSUE TITLE
Add "Ignore delimiter" option to each url or setting custom delimiter field

### ISSUE THREAD

**Issue by MurzNN** [Qwertovsky/quick_url_addon#4]
Now addon always splits the input string to separate parameters by slashes. Sometimes it's convenient, but when I really need to pass string with slashes (eg the string "/user/password/reset") - it makes this task impossible.

Can you add the option "ignore delimiter" for specific urls, or field to change default `/` delimiter to some other symbol, to fix this issue?

**Comment by Qwertovsky** [comment]
user/password/reset - I see here 3 parameters.
`https://example.com/{?}/{?}/{?}` + `user/password/reset` => `https://example.com/user/password/reset`
Don't add one more slash.`/user/password/reset` - 4 parameters.
You task is possible.

**Comment by MurzNN** [comment]
Yeah, thanks for the workaround idea! But in my case the number of slashes is not predictable (urls can have more or less slashes), and as result I've got the trailing items, if fill parts with a margin:
- `https://example.com/router/translate-path?path={?}/{?}/{?}/{?}/{?}/{?}` + `/user/password/reset` => `https://example.com/router/translate-path?path=/user/password/reset/{?}/{?}` instead of `https://example.com/router/translate-path?path=/user/password/reset`
- `https://example.com/router/translate-path?path={?}/{?}/{?}/{?}/{?}/{?}` + `/node/123` => `https://example.com/router/translate-path?path=/node/123/{?}/{?}/{?}/{?}` instead of `https://example.com/router/translate-path?path=/node/123`
